lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,89 @@
+defmodule SituationRoom.Site.Controller do
+  @moduledoc """
+  Documentation for `SituationRoom.Site.Controller`.
+  """
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+
+  plug(Plug.Parsers,
+    parsers: [:urlencoded, :multipart],
+    pass: ["text/*"]
+  )
+
+  plug(:dispatch)
+
+  # Get all sites
+  get "/" do
```

Comment:
So there may have been some misunderstanding here, but I'm not sure why we dropped all the "objects" from the routes?  The "root" route should deal with sites.  I'd expect the routes for anything related to sites to be the following under REST:

    GET /sites
    POST /sites
    GET /sites/:id 
    PUTS /sites/:id 
    DELETE /sites/:id

Reply:
I have all of the routes that are prefixed with `/site` forward to this controller. This way, I can abstract the `/site` out from the site controller. All of the routes that you have suggested are valid and reachable (Except PUTS - I have not implemented that).